package.json

```
@@ -4,7 +4,7 @@
   "description": "A CLI that uses swagger-jsdoc to generate Swagger definitions, but dereferences all json-refs for compatibility.",
   "main": "index.js",
   "scripts": {
-    "lint": "eslint-godaddy *.js \"{bin,test}/**/*\"",
+    "lint": "eslint-godaddy *.js \"{bin,test}/**/*.js\"",
```

This was added as the new `additional-api.yaml` file was making it real mad.

---

... and I just realized `bin/swagger-jsdoc-deref` doesn't have a `.js` extension. Any reason for that?

---

I don't think there was any specific reasoning there. you could use [`--ignore-pattern`](https://eslint.org/docs/user-guide/command-line-interface#-ignore-pattern) to ignore the yaml files. something like this:
```suggestion
    "lint": "eslint-godaddy --ignore-pattern \"**/*.yaml\" *.js \"{bin,test}/**/*\"",
```